artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -3946,6 +3954,14 @@
             </xsd:annotation>
          </xsd:element>
 
+         <xsd:element name="prefetch-page-bytes" type="xsd:string" default="20M" maxOccurs="1" minOccurs="0">
+            <xsd:annotation>
+               <xsd:documentation>
+                  How many bytes we are reading from paging and keeping ready in queues ready to deliver to consumers.
```

Comment:
The default could be spelled out clearer? It isnt really just 20M, but whatever max-read-page-bytes is (the default for which is also not just 20M but 2x page size...which similarly isnt clear). Similar to other comment, interaction of the two things isnt that clear either.

Similar for the messages config.